.changeset/shaggy-clouds-invent.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
I wonder if this should warrant an actual changeSet message since we are changes the level of compatibility?
Probably good to have in the change logs rather needing to look into the code.

Reply:
I would say it's a patch release .. the only actual code changes relate to a test.  The library code itself doesn't change, so it technically would continue to work with Node 14